firefly/app.py

```diff
@@ -47,6 +52,20 @@ def generate_index(self):
             }
         return help_dict
 
+    def generate_showcase(self, **kwargs):
+        functions = [
+                {'name': name, 'path': spec['path'], 'doc': spec['doc'], 'parameters': spec['parameters']}
+                for name, spec in self.generate_function_list().items()
+                ]
+        html = template.render({
+            'host_url': kwargs['host_url'],
+            'functions': functions
+            })
+        response = Response(content_type='text/html')
+        response.status = 200
+        response.text = html
+        return response
+
```

Comment:
Wouldn't `render_docs` a better name for this function?

Reply:
Agreed!